indico/modules/events/timetable/client/js/Timetable.module.scss

```
@@ -79,6 +81,15 @@
 
       &.selected {
         filter: opacity(0.9) brightness(0.95);
+        box-shadow: -0.1em 0 1em 0.2em $indico-blue;
+        margin-left: 0.2em;
+        z-index: 0;
+
+        &.parent-entry-selected {
+          box-shadow: none;
+          outline: none;
+          margin-left: 0.2em;
```

you probably meant to put a zero here?
```suggestion
          margin-left: 0;
```
otherwise this doesn't really change anything